### DISCUSSION TITLE
How to delete part of the line which is in relation

### DISCUSSION THREAD

**Discussion by macias** [openstreetmap/iD#9286] [General]
I hae a case when I need to delete some part of the line (path) because it does no exists in reality. So I split a line so I know what I would like to remove for sure. Yet, I cannot because Id Editor states this entire line is referenced by relation.

So now I have two problems:
* how I identify "my" part of the line within the relation, they are listed in such way that I cannot tell the difference
* how do I delete anything from relation -- I don't see any trash button

?

**Comment by boxer4** [comment]
Select the way/point.  iD should give a list of relations that a way/point is part of in the left bar at the bottom.  Simply trash them from the relation first - after they are no longer part of any relation, then you can remove the way/point.
